python/runtime/cudaq/algorithms/py_observe.cpp

```diff
@@ -302,6 +303,20 @@ void bindObserve(py::module &mod) {
         }
 
         // Return the vector of results, this is for observe_n
```

Comment:
nit: I think this pre-existing `// Return the vector of results, this is for observe_n` comment should be moved below your new code to be kept with the `return {result}` statement.